ecf/scripts/gdas/atmos/analysis/jgdas_atmos_analysis_wdqms.ecf

```diff
@@ -0,0 +1,63 @@
+#PBS -S /bin/bash
+#PBS -N %RUN%_atmos_analysis_wdqms_%CYC%
+#PBS -j oe
+#PBS -q %QUEUE%
+#PBS -A %PROJ%-%PROJENVIR%
+#PBS -l walltime=00:10:00
+#PBS -l select=1:mpiprocs=1:ompthreads=1:ncpus=1:mem=48GB
+#PBS -l place=vscatter:shared
+#PBS -l debug=true
+
+export model=gfs
+%include <head.h>
+%include <envir-p1.h>
+
+set -x
+
+export NET=%NET:gfs%
+export RUN=%RUN%
+export CDUMP=%RUN%
+
+############################################################
+# Load modules
+############################################################
+module load PrgEnv-intel/${PrgEnv_intel_ver}
+module load craype/${craype_ver}
+module load intel/${intel_ver}
+module load cray-mpich/${cray_mpich_ver}
+module load cray-pals/${cray_pals_ver}
+module load hdf5/${hdf5_ver}
+module load netcdf/${netcdf_ver}
+# TODO: Needs the python modules. Review the above list (hdf5 and netcdf)
```

Comment:
Leaving comment to check on this later since it's TODO.

The module list looks good and matches the base set of modules that most jobs currently load in ops on WCOSS2. Other jobs load `python/3.8.6` set via `versions/run.ver`.

Reply:
sounds good.  I will load the python module.  We will need to ensure it contains pandas and whatever else the `wdqms.py` requires.